src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -6,8 +6,9 @@ import PropTypes from 'prop-types';
 import Image from '../../../components/Image';
 import MemoIcAddCircle from '../../../assets/icons/ic_add_circle';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../../utils/constants';
-import {normalize} from '../../../utils/fonts';
+import {normalize, normalizeFontSize} from '../../../utils/fonts';
 import {CircleGradient} from '../../../components/Karma/CircleGradient';
+import dimen from '../../../utils/dimen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `normalizeFontSize` and `dimen` modules are added but they are not used anywhere in the provided code. If these modules are not used elsewhere in the file, it's better to remove them to keep the code clean and avoid unnecessary imports.
```diff
- import {normalize, normalizeFontSize} from '../../../utils/fonts';
- import dimen from '../../../utils/dimen';
+ import {normalize} from '../../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->